routes/web.php

```diff
@@ -20,11 +20,11 @@
 });
 
 Route::get('/tasks/index', [TaskController::class, 'index'])->name('tasks.index');
-Route::get('/tasks/create', [TaskController::class, 'create'])->name('tasks.create');
+Route::get('/tasks/create', [TaskController::class, 'create'])->name('tasks.create')->middleware(['admin',]);
```

Comment:
```suggestion
Route::get('/tasks/create', [TaskController::class, 'create'])->name('tasks.create')->middleware(['admin']);
```
array trên 1 hàng ko cần "," ở cuối

Reply:
Em đã sửa ạ